Keas.Mvc/Services/TeamsManager.cs

```diff
@@ -0,0 +1,192 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Keas.Core.Data;
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+using Serilog;
+
+namespace Keas.Mvc.Services
+{
+    public interface ITeamsManager
+    {
+        Task<string[]> GetTeamRoleNames(string slug);
+        Task<string[]> GetTeamOrAdminRoleNames(string slug);
+        Task<string[]> GetSystemRoleNames();
+        Task<Dictionary<string, string>> GetMyTeams();
+        void ClearTeamSession();

```

Comment:
still don't like this having the session as part of the name, but it's not that bad